cmd/rofl/deploy.go

```diff
@@ -114,8 +116,10 @@ var (
 
 			// Push ORC to OCI repository.
 			if deployment.OCIRepository == "" {
-				// TODO: Support default OCI repository.
-				cobra.CheckErr(fmt.Sprintf("Missing OCI repository for deployment '%s'.", deploymentName))
+				// Use default OCI repository with random name and timestamp tag.
+				deployment.OCIRepository = fmt.Sprintf(
+					"%s/%s:%d", buildRofl.DefaultOCIRegistry, uuid.New(), time.Now().Unix(),
+				)
 			}
 			fmt.Printf("Pushing ROFL app to OCI repository '%s'...\n", deployment.OCIRepository)
 
```

Comment:
Can you store the config file's `oci_repository` field at this point, if the orc was successfully stored so we avoid multiple uploads, if someone forgets to pick the offer or term.

Reply:
(You just need to run `manifest.Save()` after successful push, see below.)